s4e-backend/src/main/java/pl/cyfronet/s4e/api/SceneSearch.java

```diff
@@ -0,0 +1,19 @@
+package pl.cyfronet.s4e.api;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class SceneSearch {
```

Comment:
SceneMapping, MappedScene? It's a result of row-mapping, search implies it could be specification of search terms for example